RELEASE.md

```diff
@@ -1,3 +1,29 @@
-# Eclipse Che-Theia release process
+## Eclipse Che-Theia release process
+See [Release Che-Theia](./.github/workflows/release.yml) workflow.
 
-See [Release Che Theia](./.github/workflows/release.yml) workflow.
+## Bugfix release with the upstream Theia commit included
+To make another bugfix release of Che-Theia with the upstream Theia patch included, there're several options available.
+
+### To include all the commits made before the required patch
+- update the `build.include` file for an interested Che-Theia tag:
+  - `THEIA_COMMIT_SHA="<commit-sha>"`
+- continue the release process as usual
+
+### To include only the required upstream commit
+- clone the Theia fork https://github.com/redhat-developer/codeready-workspaces-theia
```

Comment:
what fork is it ? theia or che-theia or codeready workspaces ?

Reply:
It was the wrong link. I updated it with the correct one.